packages/nimble-components/src/card/specs/README.md

```diff
@@ -0,0 +1,116 @@
+# Nimble Card
```

Comment:
We should get the team's opinions on calling this "card" vs "container" or something else. We can always change our minds and rename it but let's make our best guess now.

My initial thought was that we might decide to have different components:
- "container" is meant for grouping related controls. One might think of it as a non-collapsible [accordion](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/accordion/specs/IxD.md).
- "card" is meant for showing summary or status information

But in a quick scan through a few other design systems I only saw cards. Maybe that (plus FAST's name choice) is enough to go with card to start with, but I'd like to do a bit more reading and see if others have thoughts

Reply:
I went through a similar thought process. We called it a "container" in the feature definition, but looking at other design systems, "card" seems like a common name for a display container grouping related content, whether that content is configuration or information. That's why I started calling it a card, although obviously I'm still open to suggestions. Container also makes sense for our use case.

The "configuration container" and "info card" use cases could be appearance variants of the same Card control. Having it as two separate components might lead to confusion about when to use which one, especially if both controls end up with their own appearance variants.

Reply:
I looked at a few more design systems today and didn't see a compelling consensus about containers and cards being different from each other. So I'm ok with picking one term and I'm ok with that term being "card".

Let's leave the thread open for other reviewers but if no one objects we can resolve this with no changes.

Reply:
Since there were no objections to this, I'm resolving this issue. Thanks for the discussion!